mlora/executor/context/lora.py

```diff
@@ -0,0 +1,103 @@
+from mlora.config import LoRAConfig
+from mlora.model.modules import LoRA
+from mlora.model.args import LinearInfo
+
+import os
+import torch
+import logging
+from typing import List, Dict
+from collections import OrderedDict
+
+from .context import TaskContext
+
+
+class LoRATaskContext(TaskContext):
+    def __init__(self, context_name: str) -> None:
+        super().__init__("lora", context_name)
+
+    def init(self, config: LoRAConfig,
+             linears_info: OrderedDict[str, LinearInfo]) -> None:
+        # init the adapter's weight
+        for linear_name, linear_info in linears_info.items():
+            target_name = linear_name.split('.')[3]
+            if target_name not in config.target_:
+                continue
+            if config.target_[target_name] is not True:
+                continue
+
+            self.adapter_model_[linear_name] = LoRA(config.name_,
+                                                    linear_info.in_dim_, linear_info.out_dim_,
+                                                    config.r_, config.alpha_, config.dropout_)
+
+        self.__init_weight()
+
+        self.device_ = "cpu"
```

Comment:
default is the cpu?

Reply:
yes, only scheduling the task in the running queue, will change the weight to the GPU for train.